app/controllers/permalinks_controller.rb

```diff
@@ -11,7 +11,7 @@ def show
     raise Discourse::NotFound unless permalink
 
     if permalink.target_url
-      redirect_to permalink.target_url, status: :moved_permanently
+      redirect_to permalink.target_url, status: :moved_permanently, allow_other_host: true
```

Comment:
Any ways this could be abused? 🤔 

Can/should we somehow "allow-list" the "other hosts" based on the permalinks?

Reply:
This is the default no?

Reply:
This was the default before no? So we were already doing it in the past, rails changed the default if I understand correctly.

Reply:
Indeed, it has always been the “normal” behavior, it was only broken recently.